nodes/ceos/ceos.go

```diff
@@ -118,6 +124,42 @@ func createCEOSFiles(node *types.NodeConfig) error {
 	cfg := filepath.Join(node.LabDir, "flash", "startup-config")
 	node.ResStartupConfig = cfg
 
+	// use interface mapping file to set the Management interface if it is provided in the binds section
+	// default is Management0
+	mgmtInterface = "Management0"
```

Comment:
the default var value has already been set on Ln 45

Reply:
@hellt Thanks for the comments. I updated my code accordingly.